packages/create-contentful-app/template/src/components/ConfigScreen.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from 'react';
+import { AppExtensionSDK } from 'contentful-ui-extensions-sdk';
+import { Heading, Form, Workbench, Paragraph } from '@contentful/forma-36-react-components';
+import { css } from 'emotion';
+
+export interface AppInstallationParameters {}
+
+interface ConfigProps {
+  sdk: AppExtensionSDK;
+}
+
+interface ConfigState {
+  parameters: AppInstallationParameters;
+}
+
+export default class Config extends Component<ConfigProps, ConfigState> {
+  constructor(props: ConfigProps) {
```

Comment:
Nit: Any reason why we do not use a functional component + hooks? Would make this file slimmer and easier to read.

Reply:
No reason tbh. It's more of preference tbh - so, it's up to the user.